### DISCUSSION TITLE
RabbitMQ implementation

### DISCUSSION THREAD

**Discussion by santisbon** [fedify-dev/fedify#82] [Ideas]
Hi, any plans for a RabbitMQ implementation of [MessageQueue](https://jsr.io/@fedify/fedify/doc/federation/~/MessageQueue) just like the one for Redis?
Thank you!

**Comment by dahlia** [comment]
Yes, of course!  Could you file an issue for it?

**Reply by santisbon** [reply]
Done! https://github.com/dahlia/fedify/issues/83

**Comment by dahlia** [comment]
Now we officially provide [`AmqpMessageQueue`](https://fedify.dev/manual/mq#amqpmessagequeue).
